livre/Lib/Configuration/VolumeRendererParameters.cpp

```diff
@@ -29,6 +29,7 @@ const std::string TEXTURECACHEMEM_PARAM = "texture-cache-mem";
 const std::string TEXTUREDATACACHEMEM_PARAM = "texturedata-cache-mem";
 const std::string MINLOD_PARAM = "min-lod";
 const std::string MAXLOD_PARAM = "max-lod";
+const std::string SAMPLESPERRAY_PARAM = "samples-per-ray";
```

Comment:
I don't agree... samples alone means nothing. For example we will have most probably samples-per-pixel in the future. I also think samples-per-ray explain very well what it is, no error possible.